app/decorators/position_decorator.rb

```diff
@@ -0,0 +1,19 @@
+class PositionDecorator < Draper::Decorator
+  delegate_all
+
+  def format_name_field
+    if object.event_type.nil?
+      h.text_field(:position, :name, value: object.name)
+    else
+      h.text_field(:position, :name, { disabled: true,
+                                       value: object.event_type.name })
+    end
+  end
+
+  def render_delete_button
+    if event_type_id.nil? && id.present? && id > 5
+      h.button_to('remove', h.position_path(self), method: :delete,
+                                                    class: 'btn btn-danger')
```

Comment:
Style/AlignHash: Align the elements of a hash literal if they span more than one line.